<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppletTestHostClientUi</class>
 <widget class="QWidget" name="AppletTestHostClientUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Host Clent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ItemsGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="m_StatusLabel">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_NetworkHostLabel">
        <property name="text">
         <string>Choose A Network Host</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_NetworkHostComboBox"/>
      </item>
      <item>
       <widget class="IdentWidget" name="m_NetworkIdentWidget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="m_Label1">
        <property name="text">
         <string>Choose A Host Service Type</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_HostTypeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="m_Label2">
        <property name="text">
         <string>Choose A Host Url</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_GroupUserStatusLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="IdentWidget" name="m_HostIdentWidget" native="true"/>
      </item>
      <item>
       <widget class="QComboBox" name="m_HostUrlComboBox"/>
      </item>
      <item>
       <widget class="HostedPluginWidget" name="m_HostedPluginidget" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_QueryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Query</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_RefreshListButton">
        <property name="text">
         <string>Refresh Group List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Group Host List From Network Host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="HostListWidget" name="m_HostListWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HostListWidget</class>
   <extends>QWidget</extends>
   <header location="global">HostListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IdentWidget</class>
   <extends>QWidget</extends>
   <header>IdentWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HostedPluginWidget</class>
   <extends>QWidget</extends>
   <header>HostedPluginWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
